python/docs/source/user_guide/pandas_on_spark/options.rst

```diff
@@ -78,7 +78,7 @@ are restored automatically when you exit the `with` block:
 
 .. code-block:: python
 
-   >>> with ks.option_context("display.max_rows", 10, "compute.max_rows", 5):
+   >>> with ps.option_context("display.max_rows", 10, "compute.max_rows", 5):
    ...    print(ks.get_option("display.max_rows"))
    ...    print(ks.get_option("compute.max_rows"))
```

Comment:
There are still `ks` here and lines 86 and 87.

Reply:
Thanks!!